dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessService.java

```diff
@@ -2356,26 +2360,40 @@ public void createTaskAndRelation(User operator,
                 }
                 updateTaskDefinition(operator, projectCode, taskNode, taskDefinition);
             }
-            taskNameAndCode.put(taskNode.getName(), taskDefinition);
+            taskDefinitionMap.put(taskNode.getName(), taskDefinition);
+        }
+        return taskDefinitionMap;
+    }
+
+    /**
+     * handle task relations
+     */
+    public int handleTaskRelation(User operator,
+                                   Long projectCode,
+                                   ProcessDefinition processDefinition,
+                                   List<TaskNode> taskNodes,
+                                   Map<String, TaskDefinition> taskDefinitionMap) {
+        if (null == processDefinition || null == taskNodes || null == taskDefinitionMap) {
+            return Constants.EXIT_CODE_FAILURE;
```

Comment:
the exit code constant is used on task execution result.
you can replace it with another constant.